.gitlab/kernel_version_testing/system_probe.yml

```diff
@@ -200,7 +200,7 @@ upload_system_probe_tests_arm64:
     - pushd $DEPENDENCIES
     # download and copy btf files
     - mkdir -p $BTF_DIR
-    - $S3_CP_CMD $S3_ARTIFACTS_URI/minimized-btfs-${BTFS_ARCH}.tar.xz $BTF_DIR/minimized-btfs.tar.xz
+    - cp $CI_PROJECT_DIR/minimized-btfs.tar.xz $BTF_DIR/minimized-btfs.tar.xz
```

Comment:
Is the `Arch` suffix removed?

Reply:
Yes, because I'm using the gitlab artifacts feature now and these jobs are arch-specific.